### PR TITLE
TEST: Fixed name of repo and move html files correctly

### PR DESCRIPTION
**Pull request by lukas-bednar**

### CODE REVIEW

**Comment by lukas-bednar** [comment]
@rmohr this is my attempt to fix last deploy of api-reference

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by lukas-bednar** [comment]
@rmohr it failed on `Couldn't push the build to github.com/kubevirt-incubator/api-reference.git:master`
I was reading around and I found that branch where you want to push to, can not be listed in "Protected branches" list, can you please check setting of api-reference repo ? I don't have permission to see it.

**Comment by rmohr** [comment]
Hm, the branch is not protected. Maybe I need to give the token more permissions.

**Comment by rmohr** [comment]
Ok, was a permission issue on the token I set up. I restarted the travis job.

**Comment by lukas-bednar** [comment]
@rmohr it wiped out original content :-) ...
but at least it got in ... but only for one job I probably need to limit it for 1.8.3 for example.

Do you have any idea what to do when I want to preserve some content ?

File: .travis.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ deploy:
     repo: kubevirt/kubevirt
 - provider: pages
   skip_cleanup: true
-  repo: https://github.com/kubevirt-incubator/api-reference.git
+  repo: kubevirt-incubator/api-reference
   target_branch: master
   github_token: $API_REFERENCE_PUSH_TOKEN
   local_dir: hack/gen-swagger-doc/html5
```
Comment (lukas-bednar) [inline .travis.yml]: on:
    branch: master
    go: 1.8.3

**Comment by rmohr** [comment]
> @rmohr it wiped out original content :-) ...
> but at least it got in ... but only for one job I probably need to limit it for 1.8.3 for example.
> 
> Do you have any idea what to do when I want to preserve some content ?

Hm, looks like wee need a custom script as deployment step now because of this.

**Pull request merged**